docs/howto/import.md

```diff
@@ -149,6 +149,7 @@ The `lakectl ingest` command currently supports the standard `GOOGLE_APPLICATION
 
 Importing is only possible from the object storage service in which your installation stores its data. For example, if lakeFS is configured to use S3, you cannot import data from Azure.
 
+Import is available for S3, GCP, Azure and the local storage adapters. While the first 3 don't require special configurations, for security reasons you need to enable it for the local storage adapter by setting `blockstore.local.import_enabled` and specifying the allowed import paths `blockstore.local.allowed_external_prefixes` described in the [configuration](link). Since there are some differences between object-stores and file-systems in the way directories/prefixes are treated, local import is allowed only for directories.
 Although created by lakeFS, import branches behave like any other branch:
```

Comment:
Maybe add couple of line breaks so that it would be presented nicely in the website?